### ISSUE TITLE
Online/Offline updates

### ISSUE THREAD

**Issue by grantal** [reed-college/paneity#269]
When you load the tutors page, it will get the status of all of the tutors on that page. This status will not update. We may want to do that, but it will be a lot of work for something that is unlikely to be used

**Comment by marikaswanberg** [comment]
Yeah, that seems really complicated to fix. I think the way that we have it now is good enough for the amount of use that it will probably get.
